moveit_planners/ompl/ompl_interface/scripts/generate_state_database.cpp

```diff
@@ -193,7 +193,7 @@ void computeDB(const rclcpp::Node::SharedPtr& node, const planning_scene::Planni
  *       y: [0.2, 0.3]
  *       z: [0.1, 0.6]
  *     weight: 1.0
- * """
+ * '"'
```

Comment:
Is this a Python docstring embedded in a C++ comment? Should this be triple backticks or something?